### PR TITLE
Fix context-menu

### PR DESCRIPTION
**Pull request by btzr-io**
### Changes

- Can't right click > paste into description on publish and textarea #1664 
- External fix  https://github.com/RIP21/react-simplemde-editor/pull/68 

### Todo
- [x] Fix textarea contex-menu.
- [x] Fix markdown editor context-menu.
- [x] Fix external issue RIP21/react-simplemde-editor#67

### CODE REVIEW

**Comment by btzr-io** [comment]
@seanyesmunt You can merge this if you want, not sure how long would take to review RIP21/react-simplemde-editor#68

**Comment by btzr-io** [comment]
@seanyesmunt It's ready for a review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
    * Incorrect markdown preview on url with parentheses ([#1570](https://github.com/lbryio/lbry-app/issues/1570))
    * Fix Linux upgrade path and add manual installation note ([#1606](https://github.com/lbryio/lbry-app/issues/1606))
    * Fix can type in unfocused fields while publishing without selecting file ([#1456](https://github.com/lbryio/lbry-app/issues/1456))
+   * Fix can't right click > paste into description on publish ([#1664](https://github.com/lbryio/lbry-app/1664))
```
Comment (seanyesmunt) [inline CHANGELOG.md]: This will need to be in a new section under `0.22.1`

File: src/renderer/util/contextMenu.js [hunk 2]
```diff
@@ -61,6 +61,30 @@ export function openContextMenu(event, templates = []) {
   injectDevelopmentTemplate(event, templates);
   remote.Menu.buildFromTemplate(templates).popup();
 }
+
+export function openEditorMenu(event, codeMirror) {
```
Comment (seanyesmunt) [inline src/renderer/util/contextMenu.js]: It might be helpful to add a comment above this function describing its use for the markdown editor

**Pull request merged**